database/v2/schemas/users.model.js

```diff
@@ -0,0 +1,57 @@
+module.exports = (sequalize, type) => {
+  const MODEL = sequalize.define(
+    'Users',
+    {
+      email: {
+        type: type.STRING(64),
+        allowNull: false,
+      },
+      password: {
+        type: type.STRING(100),
+      },
+      id: {
```

Comment:
what sense of using uuid as id for users and other entities ?

Reply:
Fixed